CHANGELOG.rst

```diff
@@ -18,6 +18,7 @@ CHANGELOG
 * build: added pylint
 * build: upgrade docker-compose to 1.23
 * enhancement: Frameworks: update warning for not setting framework_version as we aren't planning a breaking change anymore
+* feature: Estimator: add script mode and Python 3 support for TensorFlow
```

Comment:
make sure this is in the correct changelog entry. maybe even warrants a bigger version bump?

Reply:
We can make this decision tomorrow with the pr to bump version?

Reply:
sure